### PR TITLE
No /apis/<group>/<version> endpoint for empty groups

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

**Comment by deads2k** [comment]
This seems weird.  How did we get to this point?

File: staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go [hunk 1]
```diff
@@ -307,6 +307,10 @@ func (s *GenericAPIServer) installAPIResources(apiPrefix string, apiGroupInfo *A
 		}
 
 		apiGroupVersion := s.getAPIGroupVersion(apiGroupInfo, groupVersion, apiPrefix)
+		if len(apiGroupVersion.Storage) == 0 {
+			continue
```
Comment (deads2k) [inline staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go]: how about a glog like above so we know its happening.

**Comment by deads2k** [comment]
nit, then lgtm.

File: staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go (hunk 1 above)
Comment (sttts) [inline staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go]: `apiGroupInfo.GroupMeta.GroupVersions` always has all versions, even if some are disabled. Compare the NewREST code.

Comment (sttts) [inline staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go]: The 4 lines above do the same, merged in https://github.com/kubernetes/kubernetes/pull/42911/files#diff-8c3b369a56b450db4f2c3715bec58edcR313. So we can drop this PR.

**Comment by sttts** [comment]
Duplicate of https://github.com/kubernetes/kubernetes/pull/42911.

**Pull request closed without merging**